### PR TITLE
added old scripts (issue #1)

### PR DESCRIPTION
**Pull request by ploedige**
Fixes issue #1 .

## Changes proposed:

- add scripts from the old irl_polymetis repository that are used to start the polymetis servers for our robots and grippers to the scripts folder of the polymetis fork

### CODE REVIEW

File: scripts/start_robot_201.sh
Comment (LiXiling) [inline scripts/start_robot_201.sh:1]: is the rest_pose specific to the kitchen robot? Maybe add a comment.

also: we check for / auto activate a conda env named "polymetis". I guess this is specific to our hardware setup. but maybe this is a good chance to check if the README mentions somewhere, that we want our env to be called `polymetis`.

**Review by LiXiling — CHANGES_REQUESTED** [review]
I think mainly a comment for the rest pose in 201. Then i will approve the PR.

**Comment by ploedige** [comment]
> Also, a little bit more detail in the PR description would be appreciated :D

Added more information to the PR description

**Comment by ploedige** [comment]
Removed the initial pose. Since robot 201 is not part of the kitchen environment, there is no reason for the initial pose.

Regarding the documentation:
Starting the server is not mentioned Readme at all. This should be part of  issue #3.

Also:
The start scripts should be unified into one script that accesses the hydra configuration file for the robots.
That is how I have done it in my thesis program. For this another issue should be added

**Review by LiXiling — APPROVED** [review]

**Pull request merged**